File: swagger/models/get_waitlist_entries_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetWaitlistEntriesResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		waitlist_entries
		Contains information about the waiting list entries.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'WaitlistEntries': 'waitlist_entries',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'waitlist_entries': 'WaitlistEntries',
		}

		'PaginationResponse': ('pagination_response', PaginationResponse),
	waitlist_entries = None

File: swagger/Models/GetWaitlistEntriesResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetWaitlistEntriesResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about the waiting list entries.
	 * 
	 * @var WaitlistEntry[] 
	 */
	public $waitlistEntries;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'WaitlistEntries' => ['waitlistEntries', 'array', WaitlistEntry::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'waitlistEntries' => ['WaitlistEntries', 'array', WaitlistEntry::class],
			];
	}
}
